cmake/linker/ld/target.cmake

```
@@ -157,7 +157,7 @@ macro(toolchain_linker_finalize)
   string(REPLACE ";" " " zephyr_std_libs "${zephyr_std_libs}")
 
   set(link_libraries "<LINK_FLAGS> <OBJECTS> -o <TARGET> <LINK_LIBRARIES> ${zephyr_std_libs}")
-  set(common_link "<LINK_FLAGS> ${link_libraries}")
+  set(common_link "${link_libraries}")
```

I believe the `<LINK_FLAGS>`  should be restored here.

The intention with the `link_libraries` and `common_link` is to keep those separated so that the `cpp_link` can be overruled when `crtbegin` `crtend` are needed, and only have `link_libraries` inside those flags.

```suggestion
  set(common_link "<LINK_FLAGS> ${link_libraries}")
```

---

you're right, makes more sense, I updated in my latest push, thank you !